airbyte-webapp/src/components/connectorBuilder/Builder/StreamConfigView.tsx

```diff
@@ -0,0 +1,35 @@
+import { BuilderCard } from "./BuilderCard";
+import { BuilderField } from "./BuilderField";
+
+interface StreamConfigViewProps {
+  streamNum: number;
+}
+
+export const StreamConfigView: React.FC<StreamConfigViewProps> = ({ streamNum }) => {
+  const streamPath = (path: string) => `streams[${streamNum}].${path}`;
+
+  return (
+    <BuilderCard>
```

Comment:
various i18ns

Reply:
Thanks for calling these out. I originally didn't do i18n for the BuilderField strings because I wanted to keep them easy to maintain and add to in the future. If we put them all behind i18n, then every time we add a new field we would also need to update the i18n strings and point it to there which sounded cumbersome to me for a process that we will need to do a lot.

But since we are i18ning all of our other text in the webapp, maybe we should just bite the bullet and do it here as well

Reply:
I thought about this a bit more - I feel I should fix all of the i18n issues in this PR _except_ for the labels and descriptions on the BuilderField components. My reasoning is that once the connector manifest JSON schema is decoupled from the python classes, we should be able to capture the titles and descriptions for all of the various fields in the JSON schema, and at that point we can pull those into the UI.

So it feels like a bit of a waste of time to migrate all of these to the en.json file, and require any new BuilderFields being added to also update that i18n file and point to it every time we add a new field. Thoughts?

Reply:
If we plan to pull in this copy from the schema in the near term I agree it doesn’t make sense to i18n-ify them right now. If possible we should add it as a task to the appropriate issue / create an issue about it as this is one of the things that’s easily missed

Reply:
Done: https://github.com/airbytehq/airbyte/issues/20271